pages/donate.js

```diff
@@ -428,3 +477,13 @@ export default function Donate() {
     </Box>
   )
 }
+
+export async function getServerSideProps(context) {
```

Comment:
Does this need to be here? Having a `getServerSideProps` function disables [static optimization](https://nextjs.org/docs/advanced-features/automatic-static-optimization)

Reply:
+1 here